src/twisted/web/test/test_distrib.py

```diff
@@ -284,22 +288,22 @@ def cbRendered(ignored):
             self.assertEqual(len(errors), 1)
             # The error page is rendered as HTML.
             expected = [
-                '',
-                '<html>',
-                '  <head><title>500 - Server Connection Lost</title></head>',
-                '  <body>',
-                '    <h1>Server Connection Lost</h1>',
-                '    <p>Connection to distributed server lost:'
-                    '<pre>'
-                    '[Failure instance: Traceback from remote host -- '
-                'twisted.spread.flavors.NoSuchMethod: '
-                    'No such method: remote_request',
-                ']</pre></p>',
-                '  </body>',
-                '</html>',
-                ''
+                b'',
```

Comment:
For future reference: `textwrap.dedent` is a nicer way to inline stub data strings like this.